src/main/java/emt/init/EMTResearches.java

```diff
@@ -648,6 +648,17 @@ public static void addResearch() {
                             .setPages(new ResearchPage(""), new ResearchPage(EMTRecipes.chargeFocus));
         }
 
+        if (!EMTConfigHandler.maintenanceFocusResearch) {
+            new EMTResearchItem(
+                    "Wand Focus: Maintenance",
+                    EMTResearchAspects.maintenanceFocus,
+                    7,
+                    4,
+                    4,
+                    new ItemStack(EMTItems.maintenanceFocus)).setParents("Wand Focus: Charging").setConcealed()
```

Comment:
Fair warning on leaving it concealed, this can cause issues in it actually revealing if not set up properly, I'm sure you have though, but fair warning if it just doesn't appear, see the SuperTiC bug that caused people to not be able to unlock kami for nearly a year

Reply:
Not telling you to change it, just a word to the wise

Reply:
I will manually research this to double check that it does.